python/podio/frame.py

```diff
@@ -10,49 +10,52 @@
 from ROOT import podio  # noqa: E402 # pylint: disable=wrong-import-position
 
 
-def _determine_supported_parameter_types(lang):
+def _determine_supported_parameter_types():
   """Determine the supported types for the parameters.
 
-  Args:
-      lang (str): Language for which the type names should be returned. Either
-          'c++' or 'py'.
-
   Returns:
-      tuple (str): the tuple with the string representation of all **c++**
-          classes that are supported
+      tuple(tuple(str, str)): the tuple with the string representation of all
```

Comment:
This reminded me that maybe we could include at some point type hints? They are already there in the comments. And then use linters that make use of this information